website/docs/components/stepper/nav/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,10 @@
+## Anatomy
+
+![Stepper Nav Anatomy](/assets/components/stepper/nav/stepper-nav-anatomy.png =257x*)
```

Comment:
The images should be appropriately sized in Figma before export that you shouldn't need "=257x*"